duckduckgo/macos-browser
# Send success pixel on successful data import

Task/Issue URL: https://app.asana.com/0/1203822806345703/1207768516988587/f

**Description**:

During [✓ Postmortem: Deduplicate passwords on import](https://app.asana.com/0/0/1207598056251596) it was mentioned that having insight on the rate of import success vs failure would be useful.

We already have Pixels for failures but cannot calculate a success rate from that alone.

This adds import success Pixels with source and make sure the failure Pixels and their source parameter are being sent properly.

**Steps to test this PR**:
1. Run this with the debugger attached
2. Perform password / bookmark / favicon imports from all the import sources (or at least a sensible subset)
3. Check that, on successful import, the pixel: m_mac_data-import-succeeded_{action}_{source} is sent



**Definition of Done**:


---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
